src/OrchardCore.Modules/OrchardCore.Users/Views/UserMenuItems-Profile.cshtml

```diff
@@ -5,15 +5,15 @@
 @if (await AuthorizationService.AuthorizeAsync(User, CommonPermissions.EditOwnUser))
 {
     <li>
-        <a class="dropdown-item" asp-area="OrchardCore.Users" asp-action="Edit" asp-controller="Admin" asp-route-returnUrl="@FullRequestPath">
```

Comment:
Shouldn't in this case (and only in this, when clicking this link) the `returnUrl` be the profile editor itself? So, you save it, then it redirects to itself with a notification about the successful update. Because now it redirects to the users list, which is not really expected either (and the user might not have permission to see it).

Reply:
When you click on a profile, there should be no auto-redirecting. When you click edit user from the Users UI, we still have a return URL to send the user back to the user's list. In this PR, we are just removing the return url from the profile link so that the user is not auto-redirected back to the last page they were on before then clicked on the profile link.

Reply:
There is auto-redirecting with this PR though. Save the profile, and you're redirected to the Users list.

Reply:
I see what you are saying. I fixed it.